spec/models/test_track/assignment_spec.rb

```diff
@@ -48,4 +48,18 @@
       end
     end
   end
+
+  describe "#unsynced?" do
+    it "returns true" do
+      expect(subject.unsynced?).to eq true
+    end
+
+    context "for a feature gate" do
+      subject { described_class.new(visitor: visitor, split_name: :feature_enabled) }
+
+      it "also returns true" do
+        expect(subject.unsynced?).to eq true
+      end
+    end
+  end
```

Comment:
Keeping this test around for clarity?

Reply:
yeah, now that we've decided that `unsynced?` means something else, it felt good to leave this test to lock it down. I also discovered that the fake assignment does leverage this to return false, so everything was getting gnarly.